pulp.spec

```diff
@@ -815,6 +815,13 @@ Requires(postun): /usr/sbin/semodule
 %description    selinux
 SELinux policy for Pulp's components
 
+%pre selinux
+# Record old version,if any, so we can limit which restorecon statement are executed later
+test -e %{_localstatedir}/lib/rpm-state/%{name} || mkdir -p %{_localstatedir}/lib/rpm-state/%{name}
```

Comment:
When the mkdir runs will it give it the right permissions and ownership? Maybe use `-m` on mkdir